### PR TITLE
sbp-dxd-m3

### PR DESCRIPTION
**Pull request by stuart-noumena**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Security Policy
```
Comment (ville-noumena) [inline SECURITY.md:1]: I think a space is needed here, it is not visualising correctly in Github viewer.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      ACCOUNT_HASH_SBP: e5e3607a0925a715429f08d12e82bcf8f996b8113857034fff8a66a067ddd3b3
+      PRIVATE_KEY_USER_1: MC4CAQAwBQYDK2VwBCIEIDUq8MTGOWZ9XHMBeGPFElvwULGUg9aj5maPTv3YKiHd
+      PUBLIC_KEY_USER_1: MCowBQYDK2VwAyEAxxCaPVAtYtgvK4FsuYVi82xtN4+hyd81vWACqJhrGM8=
+      ACCOUNT_HASH_USER_1: d6c48c5c2378e30905bb7916cde5fe5fcae082a0e3eb5093d7435cc7c800bee7
+      PRIVATE_KEY_USER_2: MC4CAQAwBQYDK2VwBCIEIO9We+5WrQPfz3YiVBRPArwOKqH+QuYZRoZ5xdfkMs7p
+      PUBLIC_KEY_USER_2: MCowBQYDK2VwAyEAMPaSs8laS9yWr4u9y3OUqf/l26f+72O9QYUMUPyk8aU=
+      ACCOUNT_HASH_USER_2: 12d91f30f0bc5dbc7d859040f1bd54a9cc2f76fdfa1bee42b0ca92eeed312eb2
+      PRIVATE_KEY_USER_3: MC4CAQAwBQYDK2VwBCIEINHPsohLk/Ijt59+rboz0YB0nm+EYWcnoVDHZxIH+UY7
+      PUBLIC_KEY_USER_3: MCowBQYDK2VwAyEAP7NLRp1Ll7MDyspv20jT/nBmGYrBKY49mvQDjxziY5w=
+      ACCOUNT_HASH_USER_3: 4af7048197bc3f13f8be2d705335f9a291a0905cb0d2505d87962e8ed5aa5dab
```
Comment (ville-noumena) [inline docker-compose.yml:36]: Are these OK??

**Review by ville-noumena — CHANGES_REQUESTED** [review]
Left few comments

**Review by vince-noumena — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (stuart-noumena) [reply docker-compose.yml:36]: Yes. These are credentials in casper's test environment (https://testnet.cspr.live/), so they're really test accounts with no real value.  In a real system we'd either collect these via the frontend and keep them securely, or we could make users enter their credentials when needed.

**Review by ville-noumena — DISMISSED** [review]
For me this is ok.

File: SECURITY.md (hunk 1 above)
Comment (stuart-noumena) [reply SECURITY.md]: fixed it.

Comment (stuart-noumena) [reply SECURITY.md]: fixed it.

**Review by vince-noumena — APPROVED** [review]

**Pull request merged**